dictionary-octopus.txt

```diff
@@ -357,6 +357,7 @@ reprovisioned
 reprovisioning
 resolv.conf
 ResolvConfPath
+RHEV
```

Comment:
What's a RHEV?

Reply:
Red Hat Enterprise Virtualization. It's mentioned in the existing docs, but never got captured in spell check previously.